packages/kbn-apm-synthtrace-client/src/lib/infra/container.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable max-classes-per-file */
+import { ContainerAsset } from '../assets';
+import { Entity, Fields } from '../entity';
+import { Serializable } from '../serializable';
+
+interface ContainerDocument extends Fields {
+  'container.id': string;
+  'kubernetes.pod.uid': string;
+  'kubernetes.node.name': string;
+}
+
+export class Container extends Entity<ContainerDocument> {
+  metrics() {
+    return new ContainerMetrics({
+      ...this.fields,
+      'kubernetes.container.cpu.usage.limit.pct': 46,
```

Comment:
Generally "random" numbers are almost always a sign that something's wrong, but you probably have a good reason for hard coding this

Reply:
Agreed, I've added these metric fields as placeholders for extensions when we really need metrics data.
These documents mainly include the `host.hostname` type fields that Asset Manager needs at this point.